my-app/src/Components/About/About.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Mypic from '../../images/mypic.jpeg';
+
+export default function About() {
+  return (
+    <section className="about">
+      <div className="about__container">
+        <h1 className="about__name">About the Author</h1>
+        <div className="about__info">
+          <img className="about__img" src={Mypic} />
```

Comment:
Every tag `img` should have a normal attribute `alt` to show the `alternative` text if the image is not uploaded.